app/views/events/tags.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render 'events', events: @events %>
```

Comment:
Hey, what you think about use `<%= render @events %> `to avoid be a little redundant?